src/createServer.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { router: usersRouter } = require('./routes/users');
+const { router: expensesRouter } = require('./routes/expenses');
+const { clearUsers } = require('./services/users');
+const { clearExpenses } = require('./services/expenses');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  app.use(cors());
+  app.use(express.json());
+  app.use('/users', usersRouter);
+  app.use('/expenses', expensesRouter);
+
+  clearUsers();
```

Comment:
Consider moving 'clearUsers()' and 'clearExpenses()' to a separate function, such as 'initializeData()', and call it from within 'createServer()'. This way, you separate the responsibility of data initialization from server creation.